crm_timesheet/views/crm_lead_view.xml

```diff
@@ -0,0 +1,177 @@
+        <field name="arch" type="xml">
+            <list position="inside">
+                <field name="show_time_control" invisible="1" />
+                <button
+                    name="button_start_work"
+                    string="Start work"
+                    tabindex="-1"
+                    type="object"
+                    icon="fa-play-circle text-success"
+                    invisible="show_time_control != 'start'"
+                    class="oe_stat_button"
+                    groups="hr_timesheet.group_hr_timesheet_user"
+                />
+                <button
+                    name="button_end_work"
+                    string="Stop work"
+                    tabindex="-1"
+                    type="object"
+                    icon="fa-stop-circle text-warning"
+                    invisible="show_time_control != 'stop'"
+                    class="oe_stat_button"
+                    groups="hr_timesheet.group_hr_timesheet_user"
+                />
+            </list>
+        </field>
+    </record>
+    <!-- Opportunities -->
+    <record id="crm_case_kanban_view_leads" model="ir.ui.view">
+        <field name="name">Add timesheet time controls to opportunity kanban</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.crm_case_kanban_view_leads" />
+        <field name="arch" type="xml">
+            <xpath expr="//footer" position="inside">
+                <field name="show_time_control" invisible="1" />
+                <a
+                    name="button_start_work"
+                    invisible="show_time_control != 'start'"
+                    class="o_kanban_inline_block text-success"
+                    string="Start work"
+                    tabindex="-1"
+                    type="object"
+                    groups="hr_timesheet.group_hr_timesheet_user"
+                >
+                    <i class="fa fa-lg fa-play-circle" title="Start" />
+                </a>
+                <a
+                    name="button_end_work"
+                    invisible="show_time_control != 'stop'"
+                    class="o_kanban_inline_block text-warning"
+                    string="Stop work"
+                    tabindex="-1"
+                    type="object"
+                    groups="hr_timesheet.group_hr_timesheet_user"
+                >
+                    <i class="fa fa-lg fa-stop-circle" title="Stop" />
+                </a>
+            </xpath>
+        </field>
+    </record>
+    <record id="crm_case_tree_view_oppor" model="ir.ui.view">
+        <field name="name">Add timesheet time controls to opportunity tree</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.crm_case_tree_view_oppor" />
+        <field name="arch" type="xml">
+            <list position="inside">
+                <field name="show_time_control" invisible="1" />
```

Comment:
@jdebetaz like for hr_timesheet_view.xml, we must change `invisible` to `column_invisible`

```suggestion
                <field name="show_time_control" column_invisible="1" />
```